protobuf/src/main/java/io/grpc/protobuf/StatusProto.java

```diff
@@ -160,10 +160,12 @@ private static Metadata toMetadata(com.google.rpc.Status statusProto, Metadata m
   private static com.google.rpc.Status toStatusProto(Status status, Metadata trailers) {
     if (trailers != null) {
       com.google.rpc.Status statusProto = trailers.get(STATUS_DETAILS_KEY);
-      checkArgument(
-          status.getCode().value() == statusProto.getCode(),
-          "com.google.rpc.Status code must match gRPC status code");
-      return statusProto;
+      if (statusProto != null) {
+        checkArgument(
+                status.getCode().value() == statusProto.getCode(),
```

Comment:
I can do this when I'm at a computer later tonight. I think the +8 was the result of running google format, and allowed by the style guide when there is more than two continuation lines.